pl_examples/loop_examples/kfold.py

```diff
@@ -106,6 +104,11 @@ def val_dataloader(self) -> DataLoader:
     def test_dataloader(self) -> DataLoader:
         return DataLoader(self.test_dataset)
 
+    def __new__(cls, *args, **kwargs):
+        obj = object.__new__(cls)
+        LightningDataModule.__init__(obj, *args, **kwargs)
```

Comment:
this is required to make sure base model is initialized properly since we are using dataclass here.

Reply:
done!